neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/impl/Neo4jIndexSpec.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.ogm.datastore.neo4j.impl;
```

Comment:
Maybe you can move this class from *.neo4j.impl to *.neo4j.index.impl to be consistent with the same class of MongoDB?

Reply:
sure, I moved the tests but for got the main class 😃